dags/webcompat_kb.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from utils.gcp import gke_command
+from utils.tags import Tag
+
+DOCS = """
+### Bugzilla to BigQuery import
+
+#### Description
+
+Runs a Docker image that fetches bugzilla bugs from
+Web Compatibility > Knowledge Base component, as well as their core
+bugs dependencies and breakage reports and stores them in BQ.
+
+The container is defined in
+[docker-etl](https://github.com/mozilla/docker-etl/tree/main/jobs/webcompat-kb)
+
+#### Owner
+
+kberezina@mozilla.com
+"""
+
+default_args = {
+    "owner": "kberezina@mozilla.com",
+    "email": ["kberezina@mozilla.com"],
```

Comment:
It might be good to add a distribution list here so that we see the errors even if you're on vacation. I'm not sure if you can add `webcompat-internal@mozilla.org` to that, as Google might block emails from senders outside the org - so maybe we'll need a dedicated distribution list for that (and our other jobs that'll follow)

Reply:
Thanks! I've added the email to the list.